mirrulations-client/src/mirrclient/disk_saver.py

```diff
@@ -48,6 +48,14 @@ def save_binary(self, path, data):
             file.close()
             print(f'Wrote binary to Disk: {path}')
 
+    def save_text(self, path, data):
```

Comment:
Should this be named save_extracted_text? It may be confused with json or other text files in the next classes.

Reply:
Rather it may make more sense to generalize the logging print statement inside the function and allow it to be used for any/all text writing.

Reply:
Ok i'll update this.